crates/autopilot/src/domain/auction/mod.rs

```diff
@@ -22,3 +22,35 @@ pub struct AuctionWithId {
     pub id: Id,
     pub auction: Auction,
 }
+
+/// The price of a token in wei. This represents how much wei is needed to buy
+/// 10**18 of another token.
+#[derive(Debug, Clone, Copy)]
+pub struct Price(pub eth::Ether);
```

Comment:
does the inner value needs to be public?

Reply:
fixed